opal/include/opal_config_bottom.h

```diff
@@ -568,6 +569,17 @@ typedef struct {
 } opal_short_float_complex_t;
 #    endif
 
+/* gcc 13 does not define SSIZE_MAX as required by the POSIX standard.
+ * As a workaround we define ours.
```

Comment:
Actually, could you fix this comment so that we don't have this stored in history?  It's not really a gcc issue; it's a platform issue (e.g., homebrew gcc 13 doesn't seem to have this problem).